src/c/core/session/stream/output_reliable_stream.c

```diff
@@ -163,35 +160,19 @@ bool uxr_next_reliable_nack_buffer_to_send(uxrOutputReliableStream* stream, uint
     return it_updated;
 }
 
-void uxr_buffer_heartbeat(const uxrOutputReliableStream* stream, ucdrBuffer* ub)
+void uxr_buffer_heartbeat(uint8_t stream_id, const uxrOutputReliableStream* stream, ucdrBuffer* ub)
```

Comment:
Same reason as InputReliableStream. The entity should be the first argument.

Reply:
I like your style. Changed!